src/components/facets/components/PriceSlider.jsx

```diff
@@ -52,8 +52,8 @@ function PriceSlider(props) {
   // Reset function of refinement with condition and refine
   useEffect(() => {
     if (maxSlider - 2 <= minSlider) {
-      setMaxSlider(minSlider + 2)
     } else {
+      isNaN(maxSlider) && setMaxSlider(minSlider)
```

Comment:
Can you add a comment to make clear why this line is here?

Reply:
👍  done @matthew1809